lbrynet/conf.py

```diff
@@ -624,7 +631,5 @@ def initialize_settings(load_conf_file=True):
     if settings is None:
         settings = Config(FIXED_SETTINGS, ADJUSTABLE_SETTINGS,
                           environment=get_default_env())
-        settings.installation_id = settings.get_installation_id()
-        settings.node_id = settings.get_node_id()
```

Comment:
instead of adding `initialize_post_conf_load` you can just move these two lines to be below `settings.load_conf_file_settings()`

Reply:
right now there are at least two spots where initialize_post_conf_load() is invoked, therefore we should copy those two lines in both spots..I believed having a function could make the code slimmer.

On top of that, if in the future we should have more fields that require the config to be loaded first, we can simply add more line to that function while keeping the code compact.
Last thing: having them in a function with a proper name makes it clear "why" we are doing that (I know hat also a comment would work..but yeah).

Do you think I should still remove the function?